plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/startup/CodeWhispererProjectStartupActivity.kt

```diff
@@ -45,12 +45,10 @@ class CodeWhispererProjectStartupActivity : StartupActivity.DumbAware {
         val actionManager = CodeWhispererExplorerActionManager.getInstance()
         val scanManager = CodeWhispererCodeScanManager.getInstance(project)
         actionManager.setMonthlyQuotaForCodeScansExceeded(false)
+        // Setting up listeners for Auto File Code Scan triggers.
```

Comment:
nit: Does this comment need to be updated? It's not only for auto scan triggers

Reply:
ah yes...will update the comment.